runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1196,7 +1197,8 @@ TR::CompilationInfo::CompilationInfo(J9JITConfig *jitConfig) :
    _lastCompThreadID(0),
    _lastDiagnosticTheadID(0),
    _lastAllocatedCompThreadID(0),
-   _arrayOfCompilationInfoPerThread(NULL)
+   _arrayOfCompilationInfoPerThread(NULL),
+   _transientClassLoaders(NULL)
```

Comment:
Ideally, the newly added monitor needs to be initialized with NULL as well. I say ideally because I remember we zero out the entire CompilationInfo object.